web/styles/zulip.css

```diff
@@ -3295,3 +3295,86 @@ select.invite-as {
         padding: 5px;
     }
 }
+
+#add-poll-modal {
```

Comment:
Does it make better sense to put this in `modal.css`? I think our goal should be to have fewer component-specific things in `zulip.css`.

Reply:
Seems like hot reloading doesn't work for `modal.css` -- I need to reload to have the changes apply :thinking: